apps/laboratory/src/components/Wagmi/WagmiPurchaseDonutAsyncPermissionsTest.tsx

```diff
@@ -1,19 +1,20 @@
+/* eslint-disable capitalized-comments */
+/* eslint-disable multiline-comment-style */
```

Comment:
we should remove this after cleaning up the comment

Reply:
removed